marshaler_test.go

```diff
@@ -83,25 +87,35 @@ also = 'that'
 a = 'test'`,
 		},
 		{
-			desc: "simple string array",
+			desc: "simple string slice",
```

Comment:
I guess it jumped to me because I looked from the go perspective... there was no linter, I thought I was smart nitpicking but I now realize it's an array from the toml perspective!